plugins/inputs/system/system.go

```diff
@@ -49,6 +49,11 @@ func (s *SystemStats) Gather(acc telegraf.Accumulator) error {
 	}
 
 	users, err := host.Users()
+	s.Log.Debugf("Found %i number of users", len(users))
+	if err != nil {
+		s.Log.Debugf("Oops error, ", err.Error())
```

Comment:
Shouldn't this be a `s.Log.Errorf` then? TBH the whole error checking below is _strange_ to say the least. Why are those _errors_ only visible if you turn on debugging?

Reply:
heh - I completely forgot about this. I should have market is a draft. I put this up only to get a version of telegraf with those two extra log messages. I'm going to close this as I didn't actually mean to merge it in.